configs/indigo.properties

```diff
@@ -0,0 +1,21 @@
+RUNNER=distribute
+FRAMEWORK=cucumber
+RUNNER_LEVEL=methods
+APP_NAME=Indigo
+APPLITOOLS_CONFIGURATION=./configs/applitools_config.json
+BASE_URL_FOR_WEB=INDIGO_URL
+BROWSER=chrome
+ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
+IS_VISUAL=false
+LOG_DIR=target
+LOG_PROPERTIES_FILE=./src/test/resources/log4j.properties
+PARALLEL=1
+PLATFORM=android
```

Comment:
If you are implementing for web, then change the PLATFORM=web here. That way you do not need to send this parameter everytime